packages/api/src/external/carequality/index.ts

```diff
@@ -1,7 +1,10 @@
 import * as patient from "./patient";
 
 const cqCommands = {
-  patient,
+  patient: {
+    discover: patient.discover,
+    remove: patient.remove,
+  },
```

Comment:
This allows VSCode to find usages of `discover` and `remove`... just using star (`*`) doesn't work.